apps/go-test-caching/README.md

```diff
@@ -0,0 +1,177 @@
+# go-test-caching
```

Comment:
Also probably going to rename this. Any recommendations?

Reply:
`go-run-conditional-tests` or `go-conditional-tests` ?

Reply:
renamed to `go-conditional-tests`